tests/unit/api/main.py

```diff
@@ -1185,29 +1188,42 @@ class Test_switch_register:
     def test_basic(self):
         """Calling switch_register should create an object in the db."""
         api.switch_register('sw0', type=MOCK_SWITCH_TYPE,
-                username="switch_user", password="switch_pass", hostname="switchname")
+                            username="switch_user",
+                            password="switch_pass",
+                            hostname="switchname")
         assert model.Switch.query.one().label == 'sw0'
 
     def test_duplicate(self):
-        """switch_register should complain if asked to make a duplicate switch."""
+        """
+        switch_register should complain if asked to make a duplicate switch.
+        """
```

Comment:
I was worried when you said that this was a violation because the pytest pep8 scan should have caught it.  I looked it up and this is what the doc says:

"_The summary line may be on the same line as the opening quotes or on the next line_"

I have no strong opinion either way, so I can go through and format them as you suggested if you want.